README.md

```diff
@@ -151,9 +151,8 @@ Verifying:
 <!-- @begin-sigstore-verify-help@ -->
 ```
 usage: sigstore verify [-h] [--certificate FILE] [--signature FILE]
-                       [--rekor-bundle FILE]
-                       [--cert-identity IDENTITY | --cert-email EMAIL]
-                       [--cert-oidc-issuer URL] [--require-rekor-offline]
+                       [--rekor-bundle FILE] --cert-identity IDENTITY
+                       --cert-oidc-issuer URL [--require-rekor-offline]
```

Comment:
Flagging for review: I've made these options mandatory, which is a significant breaking change. I can drop that to keep the changeset smaller, though.

Reply:
Yeah, I think that's because of the `os.getenv(...)` fallthrough. I'll see about fixing that, thanks!